config/locales/sv.yml

```diff
@@ -631,6 +642,19 @@ sv:
       member: Medlem
       applications: *applications
       applications_tooltip: Antal öppna ansökningar
+      expire_date: *expire_date
+
+  payments:
+    create:
+      page_title: &page_title '%{payment_type} Betalning genom: %{user_name}'
+      title: *page_title
+      something_wrong: Något gick fel - kontakta systemadministratören på medlem@sverigeshundforetagare.se
+
```

Comment:
Same as above: Shouldn't this message get its e-mail address through a placeholder instead of using a hard-coded one?